app/src/main/java/org/permanent/permanent/network/NetworkClient.kt

```diff
@@ -123,6 +125,7 @@ class NetworkClient(private var okHttpClient: OkHttpClient?, context: Context) {
         tagService = retrofit.create(ITagService::class.java)
         profileService = retrofit.create(IProfileService::class.java)
         storageService = retrofit.create(IStorageService::class.java)
+        legacyPlanningService = retrofitNewBaseUrl.create(ILegacyPlanningService::class.java)
```

Comment:
Maybe not right now but this Network client becomes bigger and bigger from what I see. It already has 700 lines and as we implement more features it will grow even larger. From what I see the NetworkClient is responsible of building the RequestBody. Maybe we could move this into the repository or in RequestBodyBuilder.